pkg/prometheus/agent.go

```diff
@@ -117,7 +108,7 @@ type Agent struct {
 
 	instanceFactory instanceFactory
 
-	kv kv.Client
+	ha *ha.Server
```

Comment:
why is this so funny?  i don't think we should be so flippant about our variables.